setup/static/render.js

```diff
@@ -1,12 +1,7 @@
 //API_TOKEN generator
-var chars = "0123456789abcdefghijklmnopqrstuvwxyz!@#$%^&*()ABCDEFGHIJKLMNOPQRSTUVWXYZ";
-var tokenLength = 12;
-var token = "";
-
-for (var i = 0; i <= tokenLength; i++) {
-	var randomNumber = Math.floor(Math.random() * chars.length);
-	token += chars.substring(randomNumber, randomNumber +1);
-   }
+var random_array = new Uint32Array(2);
+crypto.getRandomValues(random_array);
+var token =  random_array[0].toString() + random_array[1].toString();
```

Comment:
Is concatenating two 32bit integers a "good" api token?
Shouldn't we use something longer, like in this function?
```
function generatePassword(length) {
  const charset = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789';
  let password = '';
  const array = new Uint8Array(length);
  crypto.getRandomValues(array);
  for (let i = 0; i < length; i++) {
    password += charset[array[i] % charset.length];
  }
  return password;
}
generatePassword(32)
```

What about creating a random token on the server side (like we do for the SECRET_KEY)?

Reply:
I could do it server side. I will also change the length to 32. Although 20 should be fine IMHO. The API is rate limited after all.

Reply:
I had a look. It indeed makes sense to re-use the server side token/password generator. I will use this with length 32.